ts/src/renderer/phaser/scenes/GameScene.ts

```diff
@@ -29,18 +31,18 @@ class GameScene extends PhaserScene {
 			console.log(Phaser.Scale.Events.RESIZE, // TODO remove
 				gameSize, baseSize, displaySize, previousWidth, previousHeight);
 
-			camera.zoom *= gameSize.height / previousHeight;
-
-			/*camera.centerOn(
-				camera.scrollX + (gameSize.width - previousWidth) / 2,
-				camera.scrollY + (gameSize.height - previousHeight) / 2
-			);*/
+			if (this.zoomSize) {
+				camera.zoom = this.calculateZoom();
```

Comment:
we are directly modifying camera's zoom property here. 

on 'zoom' events we use camera.zoomTo() and a tween, but for 'resize' events we do this.

Reply:
that is correct. when we get a zoom event during game we want a smooth transition, and when window changes size or device changes orientation we want immediate change

Reply:
would `camera.setZoom()` be better?

Reply:
it would be the same